spec/index.html

```diff
@@ -536,6 +569,70 @@ <h3>RDF Documents and Syntaxes</h3>
       and different ordering of triples. While these aspects can have great
       effect on the convenience of working with the <a>RDF document</a>,
       they are not significant for its meaning.</p>
+
+    <p>The basis for <a>concrete RDF syntaxes</a> is the structure of the
+      <a href="#data-model">abstract data model</a>,
+      called the <dfn>abstract syntax</dfn>;
+      summarized in the following two tables.</p>
+    <table id="tab-graph-abstract-syntax" class="simple">
+      <caption>The abstract syntax of RDF graphs</caption>
+      <tr>
+        <th>Production</th>
+        <th>Defined as</th>
+      </tr>
+      <tr>
+        <td><a>RDF graph</a></td>
+        <td><em>a set of zero or more</em> <a>triples</a></td>
+      </tr>
+      <tr>
+        <td>triple</td>
```

Comment:
Is there a reason why you mark up "RDF graph" as a link but the rest of the entries in this column of the table are not marked up as links? That looks a bit odd.

(Same comment for the second table as well.)

Reply:
Good point! I wanted to link to what is being defined; but having all productions linked to their definitions was a bit too much, so I just left the leading ones (the ones whose structures are being defined). But those can also be linked in the table caption, which I think looks better. Done in 7d2d97fa3015f2d984a22a0f882c1c1e5dfa522c.